pkg/util/namespace/namespace.go

```diff
@@ -7,11 +7,19 @@ import (
 	"strings"
 )
 
-// IsOpenShift returns true if ns is an openshift managed namespace.
+// IsOpenShift returns true if ns is an openshift managed namespace, including the default namespace
 func IsOpenShift(ns string) bool {
 	return ns == "" ||
 		ns == "default" ||
 		ns == "openshift" ||
 		strings.HasPrefix(ns, "kube-") ||
 		strings.HasPrefix(ns, "openshift-")
```

Comment:
Now we can probably do something like this:

```suggestion
	return ns == "" || ns == "default" || IsOpenShiftSystemNamespace(ns)
```

Reply:
I was planning to do that, but then didn't because I believe the list will probably diverge further.